### DISCUSSION TITLE
Map optimization requirements

### DISCUSSION THREAD

**Discussion by blue-eric** [jheredianet/Teslamate-CustomGrafanaDashboards#94] [Ideas]
Hello, I found that this project uses Amap, which is more suitable for car enthusiasts in China. Can you now also support Amap in your project? thank you. Here is the project address: https://github.com/hipudding/teslamate

**Comment by jheredianet** [comment]
Hi, I've move this into a discussion since it's not an issue. 
Sorry but this project depends on plugins used in the official TeslaMate project for Grafana and procedures of TeslaMate. This project is not about modifying the TeslaMate core code, these are just additional dashboards for the main setup. So, your suggestion is more suitable on the main repository of TeslaMate rather than here.

**Reply by blue-eric** [reply]
Thank you for your reply, take a look at here, https://github.com/hipudding/teslamate_fix_addrs/issues/7, I pull the maintainer project, has put teslamate code changes, to fit the new dashboard, too, You need to modify or add some parameters.

**Reply by jheredianet** [reply]
As I've told you in the previous message, this is not a TeslaMate fork, this is just an extension of the grafana dashboards. You should suggest your idea in the TeslaMate repository.
